pytorch_lightning/utilities/cloud_io.py

```diff
@@ -35,10 +35,11 @@ def modern_gfile():
     file operations
     """
     tb_version = version.parse(tensorboard.version.VERSION)
-    modern_gfile = tb_version >= version.parse('2.0')
+    modern_gfile = tb_version >= version.parse("2.0")
+    return modern_gfile
 
 
-def cloud_open(path: pathlike, mode: str, newline:str = None):
+def cloud_open(path: pathlike, mode: str, newline: str = None):
     if sys.platform == "win32":
```

Comment:
```suggestion
    if platform.system() == 'Windows'
```
You can `import platform` above.